tests/Unit/fbproductTest.php

```diff
@@ -360,4 +360,216 @@ public function test_gtin_for_variable_product_unset() {
 
 		$this->assertEquals(isset($data['gtin']), false);
 	}
+
+		
+	/**
+	 * Test it gets rich text description from post meta.
+	 * @return void
+	 */
+	public function test_get_rich_text_description_from_post_meta() {
```

Comment:
"_from_post_meta" rather then saying what is the storage of the data, I think better operate at level of data input at user level, e.g. "woo product description", "woo short product description", "FB product description"